packages/config/package.json

```diff
@@ -3,10 +3,13 @@
   "version": "0.0.0-development",
   "description": "Config contains the configuration types and validation function used in the cypress electron application.",
   "private": true,
-  "main": "lib/index.js",
+  "main": "index.js",
   "scripts": {
-    "build-prod": "tsc --project .",
-    "clean": "rm lib/options.js || echo 'ok'",
+    "watch": "yarn build --watch",
```

Comment:
does this change require this watch script to be running to make configuration updates and validate changes live?

Reply:
In the first attempt, I did this wrong.

I ended up usin gulp createPackage to scaffold it. much better starting point.